src/app/core/model/Employee.ts

```diff
@@ -0,0 +1,15 @@
+import { ImageCollection } from './ImageCollection';
+
+export class Employee {
```

Comment:
use this model https://github.com/geekhub-php/theatre-front/blob/develop/src/app/core/model/employee/Employee.ts